clmm/galaxycluster.py

```diff
@@ -276,8 +276,6 @@ def compute_tangential_and_cross_components(
             ra_lens=self.ra,
             dec_lens=self.dec,
             geometry=geometry,
-            is_deltasigma=is_deltasigma,
```

Comment:
Comment on the docstrings in line 221, where it writes calls `clmm.dataops.compute_tangential_and_cross_components` with the argument `is_deltasigma`. The docstrings  should be updated.